src/code/game.c

```diff
@@ -61,26 +61,27 @@ void GameState_SetFBFilter(Gfx** gfx) {
 void func_800C4344(GameState* gameState) {
     Input* selectedInput;
     s32 hexDumpSize;
-    u16 hReg82;
+    u16 inputCompareValue;
 
-    if (HREG(80) == 0x14) {
-        __osMalloc_FreeBlockTest_Enable = HREG(82);
+    if (R_HREG_MODE == HREG_MODE_HEAP_FREE_BLOCK_TEST) {
+        __osMalloc_FreeBlockTest_Enable = R_HEAP_FREE_BLOCK_TEST_TOGGLE;
     }
 
-    if (HREG(80) == 0xC) {
-        selectedInput = &gameState->input[(u32)HREG(81) < 4U ? HREG(81) : 0];
-        hReg82 = HREG(82);
-        HREG(83) = selectedInput->cur.button;
-        HREG(84) = selectedInput->press.button;
-        HREG(85) = selectedInput->rel.stick_x;
-        HREG(86) = selectedInput->rel.stick_y;
-        HREG(87) = selectedInput->rel.stick_x;
-        HREG(88) = selectedInput->rel.stick_y;
-        HREG(89) = selectedInput->cur.stick_x;
-        HREG(90) = selectedInput->cur.stick_y;
-        HREG(93) = (selectedInput->cur.button == hReg82);
-        HREG(94) = CHECK_BTN_ALL(selectedInput->cur.button, hReg82);
-        HREG(95) = CHECK_BTN_ALL(selectedInput->press.button, hReg82);
+    if (R_HREG_MODE == HREG_MODE_INPUT_TEST) {
+        selectedInput = &gameState->input[(u32)R_INPUT_TEST_CONTROLLER_PORT < 4 ? R_INPUT_TEST_CONTROLLER_PORT : 0];
+
+        inputCompareValue = R_INPUT_TEST_COMPARE_VALUE;
+        R_INPUT_TEST_BUTTON_CUR = selectedInput->cur.button;
+        R_INPUT_TEST_BUTTON_PRESS = selectedInput->press.button;
+        R_INPUT_TEST_REL_STICK_X = selectedInput->rel.stick_x;
+        R_INPUT_TEST_REL_STICK_Y = selectedInput->rel.stick_y;
+        R_INPUT_TEST_REL_STICK_X_2 = selectedInput->rel.stick_x;
+        R_INPUT_TEST_REL_STICK_Y_2 = selectedInput->rel.stick_y;
```

Comment:
`REL_STICK` -> `ADJ_STICK`

Reply:
this feels weird deviating away from what the field is called

I understand us taking the liberty within actors and other things to name it more appropriately for its purpose, but for something as low level as an input test, I feel like they should match in name